cmd/image-builder/main.go

```diff
@@ -64,56 +46,55 @@ func main() {
 		panic(err)
 	}
 
-	if conf.GlitchTipDSN != "" {
-		err = sentry.Init(sentry.ClientOptions{
-			Dsn: conf.GlitchTipDSN,
-		})
-		if err != nil {
-			panic(err)
-		}
-	}
-
-	err = logger.ConfigLogger(logrus.StandardLogger(), conf.LogLevel)
-	if err != nil {
-		panic(err)
-	}
-	logrus.AddHook(&ctxHook{})
-
-	gitRev, err := gitRev()
+	hostname, err := os.Hostname()
 	if err != nil {
-		logrus.Warn(err.Error())
-		gitRev = "unknown"
+		hostname = "image-builder-unknown"
 	}
 
-	logrus.Infof("Starting image-builder from Git Hash: %s", gitRev)
-	logrus.Infof("Changelog: https://github.com/osbuild/image-builder/commits/%s", gitRev)
```

Comment:
All messages automatically get git sha in `build_id` so this is not necessary. The starting message repeast once more below.